pkg/controller/topology.go

```diff
@@ -39,8 +39,14 @@ import (
 	"k8s.io/klog/v2"
 )
 
+type topologyKey struct {
```

Comment:
I find it strange to call something a `Key` if it has a `Key` field inside. `topologyKeyValue`? `topologyItem`?

Reply:
Renamed to topologySegment, according to CSI spec.